modules/server/certbot/certbot.service.tmpl

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Certbot renewing ssl certificates for https
+After=docker.service,haveged.service,searchbroker.service,auth.service,mdr.service,nginx.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStartPre=/usr/bin/docker exec nginx mkdir -p /tmp/.well-known/acme-challenge
+ExecStartPre=/usr/bin/docker pull certbot/certbot:${version}
+ExecStart=/usr/bin/docker run \
+    --name=certbot \
+    --rm \
+    --network=searchbroker \
+    -v /etc/nginx/ssl:/etc/letsencrypt \
+    -v acme-challenge:/tmp \
```

Comment:
is acme-challenge a volume? were is it defined?

Reply:
Done